client/resources/scene_manager.py

```diff
@@ -58,7 +58,7 @@ def __add_scene_to_history(self, scene: Scene) -> None:
         self.scene_history.append(scene)
 
     @staticmethod
-    def __switch_to_scene(fn) -> callable:
+    def __switch_to_scene(fn) -> Callable:
```

Comment:
`Callable` is not precise enough, this mean it can take any number of argument and return Any types